src/App.jsx

```diff
@@ -29,7 +29,16 @@ const App = () => {
       }
 
       const data = await response.json();
-
+      console.log(data);
+      data.records.sort(function(objectA, objectB) {
+        if (objectA.fields.title < objectB.fields.title) {
```

Comment:
Nice, same can be achieved with localecompare prototype method

Reply:
I see. thank you